README.md

```diff
@@ -801,15 +801,13 @@ See the [AUTHORS][] file for author contact information.
 License
 -------
 
-Bedrock and all Bedrock modules are:
-
-    Copyright (c) 2011-2024 Digital Bazaar, Inc.
-
 You can use Bedrock for non-commercial purposes such as self-study, research,
-personal projects, or for evaluation purposes. Many Bedrock libraries (including
-the core) are licensed as open source under the [BSD-3-Clause](LICENSE.md). However,
-some components use a custom non-commercial source available license: see the
-[LICENSES/LicenseRef-bedrock.md](LICENSES/LicenseRef-bedrock.md) file for details.
+personal projects, or for evaluation purposes. Bedrock libraries (including
+the core) are licensed as open source under the [BSD-3-Clause](LICENSE).
+
+Some components use a custom non-commercial source available license: see the
```

Comment:
```suggestion
Some Bedrock components use a custom non-commercial source available license: see the
```

Reply:
There's newer text for this now that we're moving to the Apache-2.0 license.